packages/react/src/React.js

```diff
@@ -52,27 +51,19 @@ import {
   createElementWithValidation,
   createFactoryWithValidation,
   cloneElementWithValidation,
-  jsxWithValidation,
-  jsxWithValidationStatic,
-  jsxWithValidationDynamic,
 } from './ReactElementValidator';
 import createMutableSource from './createMutableSource';
 import ReactSharedInternals from './ReactSharedInternals';
 import createFundamental from 'shared/createFundamentalComponent';
 import createResponder from 'shared/createEventResponder';
 import createScope from 'shared/createScope';
+import {jsx, jsxs, jsxDEV} from './jsx/ReactJSX';
```

Comment:
We’re still re-exporting from the React object. I don’t think we want that. Remove these. Then remove them from all top-level `index*.js` entry points. Only add that import to `index.fb.js` and re-export them there.

Reply:
Otherwise we’ll effectively ship `React.jsx` by accident.